mapper/thin_edge_json/examples/tej_example.rs

```diff
@@ -0,0 +1,44 @@
+use chrono::offset::FixedOffset;
+use chrono::DateTime;
+use chrono::TimeZone;
+use thin_edge_json::measurement::FlatMeasurementVisitor;
+use thin_edge_json::{group::MeasurementMap, serialize::ThinEdgeJsonSerializer};
+
+fn test_timestamp() -> DateTime<FixedOffset> {
+    FixedOffset::east(5 * 3600)
+        .ymd(2021, 04, 08)
+        .and_hms(0, 0, 0)
+}
+
+fn tej_build_serialize() {
+    //Produce the TEJ from raw data
+    let mut grp_msg = MeasurementMap::new(test_timestamp());
+
+    grp_msg.timestamp(test_timestamp()).unwrap();
+    grp_msg.measurement(None, "temperature", 25.0).unwrap();
```

Comment:
It's better to have no `unwrap()` here, as the purpose of this example is to show how the API will be used.